### PR TITLE
[BackEnd] Feature for Books Image API Added

### LINKED ISSUE #188: Define API Endpoint Contract for Image Upload and Image Display
Define the API endpoint contract for how frontend will give an uploaded picture to the backend.
Define the API endpoint contract for how the backend will give the frontend and image (what endpoints will change).

### PR DESCRIPTION
**Pull request by hkder**
## Feature Description/Implementation
- Requirement 1.11 Image file handling, 
- Requirement 2.2 Book Detail (The cover art, if present, should be shown at a decently large size
here (i.e., larger than a thumbnail)
(Closes #188 )
- Requirement 2.3.2 Book Add (with the cover in thumbnail format)
- Requirement 2.3.3 For cover art, this means allowing the user to upload a new image or remove the currently associated image.
- Requirement 2.4 For cover art, this means allowing the user to upload a new image or remove the currently associated image.

## Dependencies
- Needs a image server working on the https://books-db.colab.duke.edu
- .env file must contain the SCP_HOST, SCP_USER, SCP_PASSWORD for scp protocol to work

## Everything Else
- Removing the image is not possible at the moment.

### CODE REVIEW

File: backend/books/serializers.py [hunk 1]
```diff
@@ -14,10 +14,11 @@ class BookListAddSerializer(serializers.ModelSerializer):
     """
     authors = serializers.SlugRelatedField(queryset=Author.objects.all(), many=True, slug_field='name')
     genres = serializers.SlugRelatedField(queryset=Genre.objects.all(), many=True, slug_field='name')
+    urls = serializers.StringRelatedField(many=True)
```
Comment (danielfeinblatt) [inline backend/books/serializers.py:17]: Why are we doing many urls here? Would there not be a single url to a photo?

**Review by danielfeinblatt — COMMENTED** [review]
One small question

File: backend/books/serializers.py (hunk 1 above)
Comment (hkder) [reply backend/books/serializers.py:17]: yeah this is to account for the case where multiple images might be needed to represent a book.

**Review by danielfeinblatt — APPROVED** [review]
LGTM

**Review by caseyszilagyi — APPROVED** [review]
LGTM

**Pull request merged**